app/views/users/_user.json.jbuilder

```diff
@@ -0,0 +1,2 @@
+json.extract! user, :id, :created_at, :updated_at
```

Comment:
why do you need this?

Reply:
@kth-wtag Apu, these `json.builder` files were auto generated by Rails for sending JSON responses from controller. We do not need these for this particular project. These files are removed in the future PRs.